Formula/openstackclient.rb

```diff
@@ -0,0 +1,262 @@
+class Openstackclient < Formula
+  include Language::Python::Virtualenv
+
+  desc "OpenStack Client"
+  homepage "https://openstack.org"
+  url "https://github.com/openstack/python-openstackclient.git",
+    :tag      => "5.2.0",
+    :revision => "c5719a12b5b84b2efd989030f17c0eacc9faf7ad"
```

Comment:
If it's possible we prefer to use python packages from PyPI since it's an official way to install
```suggestion
  url "https://files.pythonhosted.org/packages/9b/09/c45ba3f9436f5b26691a4b1a309c959903bda6d8a5be04eec14f32db451a/python-openstackclient-5.2.0.tar.gz"
  sha256 "0a36aea3596c201593c4b1f54e6bfd57d1bd2920b124a5751df8d05d80498d6a"
```